content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -63,6 +63,17 @@ Consider the following example for 3 monitors:
 
 This can result in the overall status being lower than the average of the individual statuses.
 
+### Exceptions for Synthetic Tests
+In certain cases there is an exception to the status calculation for monitor-based SLOs that are comprised of one grouped Synthetic Test. Synthetic Tests have special alerting conditions that will change the behavior of when the test will enter the ALERT state. These conditions include:
+
+- Wait until the groups are failing for a specified number of minutes (default: 0)
+- Wait until a specified number of the groups are failing (default: 1)
+- Retry a specified number of times before a location's test is considered a failure (default: 0)
+
+{{< img src="monitors/service_level_objectives/synthetic_alert_conditions.png" alt="synthetic test alert conditions"  >}}
+
```

Comment:
We could add a doc link to https://docs.datadoghq.com/synthetics/api_tests/?tab=httptest#alert-conditions